### ISSUE TITLE
Resizing screen causes wobbling when scrollbar appears and dissappears

### ISSUE THREAD

**Issue by emitdlr** [tiberiuzuld/angular-gridster2#669]
There seems to be an issue when resizing. Right at the point where the scrollbar appears, the screen starts to wobble. I changed overflow-y: overlay which stops it, but is this a bug within Gridster? Any workaround solutions?

![how do widgets stack_ (1)](https://user-images.githubusercontent.com/34352893/92753239-50b18a80-f34f-11ea-813e-8014d1f60661.gif)

https://drive.google.com/file/d/1Og3YLviSKodzl1HGorp1XDHGg_6vQu8n/view

**Comment by tiberiuzuld** [comment]
Hi @emitdlr ,
Yes is a bug in gridster most probably.
What version of angular and library do you use?

the problem most probably appears from the height being so close to show the scrollbar.

you have the entire layout made with this library ? 
maybe use it only for the widgets

**Comment by emitdlr** [comment]
Yes, the entire layout is used with Gridster2. Running 9.1.4 using the 9.x branch

**Comment by tiberiuzuld** [comment]
Well it is possible I fixed the issue since then in the newer versions.

But my suggestion is to use [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) if you don't need resize or drag for users.

if you need that then maybe use this library only for the main widgets which users can drag and resize so the possibility of encountering issues like this is smaller.

**Comment by emitdlr** [comment]
Thanks for your feedback, I will discuss with my team on options.

**Comment by tiberiuzuld** [comment]
This is the code which should protect against this issue
https://github.com/tiberiuzuld/angular-gridster2/blob/master/projects/angular-gridster2/src/lib/gridster.component.ts#L197

**Comment by emitdlr** [comment]
Would it be possible to pull the master to the 9.x branch? For now, I'm going to add the changed code to the 9.x branch and see if that will fix it locally for me.
